invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js

```diff
@@ -121,6 +140,10 @@ export const RecordVersionsList = ({ record, isPreview }) => {
         </Placeholder>
       </>
     )
+  ) : error ? (
+    <div className="rel-mr-1 rel-ml-1">
+      <Message negative>{error}</Message>
```

Comment:
have you considered using `ErrorMessage` component from react-invenio-forms? maybe it is worth using re-usable components for the long run